community-manager-self-care.adoc

```diff
@@ -80,47 +80,47 @@ Set realistic expectations on deliverables and communicate frequently within the
 
 By taking steps towards actively promoting and advocating for self-care, you prompt members following your lead to do the same. You also begin revealing your human side, eroding the unhealthy addumption that leaders are invulnerable to emotional dimensions of the work.
 
-=== Being human
+=== Being (imperfectly and empathetically) human
```

Comment:
My only concern about this section in general is that it advances some claims that could use supporting evidence (or at least additional reading!), perhaps available upon review from another subject-matter expert.

Reply:
Understandable, as I said in my PR that retrospectively pulling in my references I bet I have missed some. I can't cite some of the points I believe you are highlighting but I can see the ones I can, so I'll schedule time to check that and update!

But yes, it's good to highlight these so that the SME can provide their input :+1:

Reply:
All makes sense to me and I'm in agreement. Thanks for considering.

Reply:
Okay dokay, I've added the remaining citations I have. When the SME is on board, this section will be highlighted requiring additional references to help verify the content. :+1: